browser/ui/webui/new_tab_page/new_tab_page_branded_view_counter.cc

```diff
@@ -37,6 +39,25 @@ std::unique_ptr<BrandedWallpaper> GetDemoWallpaper() {
   return demo;
 }
 
+std::unique_ptr<BrandedWallpaper> GetWallpaperFromData(
+    const NTPSponsoredImagesData& data) {
+  // Validate
+  if (data.wallpaper_image_count <= 0) {
+    return nullptr;
+  }
+  auto wallpaper = std::make_unique<BrandedWallpaper>();
+  wallpaper->logo = std::make_unique<BrandedWallpaperLogo>();
+  wallpaper->logo->altText = data.logo_alt_text;
+  wallpaper->logo->companyName = data.logo_company_name;
+  wallpaper->logo->destinationUrl = data.logo_destination_url;
+  wallpaper->logo->imageUrl = "chrome://branded-wallpaper/logo.png";
+  for (int i = 1; i <= data.wallpaper_image_count; i++) {
+    wallpaper->wallpaperImageUrls.push_back(
+        "chrome://branded-wallpaper/wallpaper-" + std::to_string(i) + ".jpg");
```

Comment:
Maybe there's a constant we can use here

Reply:
Pushed one more commit https://github.com/brave/brave-core/commit/66e7943480a16e2060f396db3caecfb964d2f0ba to export constants.

Reply:
you can use constants from `components/ntp_sponsored_images/url_constants.cc`

Reply:
done